hw/ip_templates/otp_ctrl/syn/otp_ctrl_gtech_syn_cfg.hjson.tpl

```diff
@@ -6,7 +6,7 @@
   name: otp_ctrl
 
   // Fusesoc core file used for building the file list.
-  fusesoc_core: lowrisc:opentitan:top_${topname}_{name}:0.1
+  fusesoc_core: lowrisc:${topname}_ip:{name}:0.1
```

Comment:
This probably should use `instance_vlnv()`, shouldn't it?

Not that I find myself caring much about gtech synthesis scripts right now. ;)

Reply:
Good catch: done.